src/app/fdctl/config.c

```diff
@@ -729,8 +729,15 @@ config_parse( int *    pargc,
     replace( result.consensus.identity_path, "{name}", result.name );
   }
 
-  replace( result.consensus.vote_account_path, "{user}", result.user );
-  replace( result.consensus.vote_account_path, "{name}", result.name );
+  if( FD_UNLIKELY( !strcmp( result.consensus.vote_account_path, "" ) ) ) {
```

Comment:
I don't think this change is right. If no vote account is provided, it starts the validator in non-voting mode and we shouldn't default this.

Reply:
thanks for catching that, removed.